client/blocks/stripe-utils/utils.js

```diff
@@ -131,6 +163,21 @@ const updatePaymentRequest = ( {
 	} );
 };
 
+/**
+ * Utility function for updating the Stripe Payment Request object using cart data form
+ * WooCommerce.
+ *
+ * @param {StripePaymentRequest} paymentRequest - The Stripe Payment Request object.
+ * @param {Object} cart - The cart data response from the store's AJAX API.
+ */
+const updatePaymentRequestWithCart = ( paymentRequest, cart ) => {
+	paymentRequest.update( {
+		total: cart.order_data.total,
+		currency: cart.order_data.currency,
+		displayItems: cart.order_data.displayItems,
+	} );
+};
```

Comment:
`updatePaymentRequestWithCart` and `updatePaymentRequest` can be removed if you implement [this suggestion](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1545#discussion_r629821055).

There are some other unnecessary functions:
- `getTotalPaymentItem`
- `getPaymentRequest`
- `canDoPaymentRequest`
- `pluckAddress`

Reply:
Fixed in 4d23f8c50c2a632f93d3f5100c7fdf6b1e3ca896.